crates/journal/src/journal.rs

```diff
@@ -139,7 +150,7 @@ pub fn new_journal_entry(app_state: Arc<AppState>, cx: &mut WindowContext) {
 fn journal_dir(path: &str) -> Option<PathBuf> {
     let expanded_journal_dir = shellexpand::full(path) //TODO handle this better
```

Comment:
@maxbrunsfeld what do you mean by handle this better? 

I noted the `shellexpand` crate will also attempt expand `ENV` variables, in case we attempt to lookup an undefined `$V`, it will fails silently. Is this the expected behavior?